pkg/apis/operator/v1alpha1/knativeserving_lifecycle.go

```diff
@@ -31,41 +31,22 @@ func (ks *KnativeServing) GroupVersionKind() schema.GroupVersionKind {
 	return SchemeGroupVersion.WithKind(Kind)
 }
 
-func (is *KnativeServingStatus) IsReady() bool {
-	return conditions.Manage(is).IsHappy()
-}
-
-func (is *KnativeServingStatus) IsInstalled() bool {
-	return is.GetCondition(InstallSucceeded).IsTrue()
-}
-
-func (is *KnativeServingStatus) IsAvailable() bool {
-	return is.GetCondition(DeploymentsAvailable).IsTrue()
-}
-
-func (is *KnativeServingStatus) IsDeploying() bool {
-	return is.IsInstalled() && !is.IsAvailable()
-}
-
-func (is *KnativeServingStatus) IsFullySupported() bool {
-	return is.GetCondition(DependenciesInstalled).IsTrue()
-}
-
```

Comment:
I liked these. :(

Reply:
They are not used anywhere though and probably shouldn't be either :). I think we used to use these to gate reconciles which we shouldn't do.